src/utils/dom.js

```diff
@@ -63,7 +63,7 @@ const addListenerMulti = (element, eventNames, listener) => {
         element.addEventListener(events[i], listener, false);
     }
 };
-export const createElement = (tag, parent, eventNames, event, content) => {
+export const createElement = (tag, parent, eventNames, event, content, persianDigits) => {
```

Comment:
createElement not clean to use toPersianDigitsIfNeeded 
plz remove persianDigits parameter and convert number before call this method

Reply:
src/datePickerDraw.js:113 is good 
src/datePickerDraw.js:194 is bad

Reply:
Should be good now!